manifests/state-nic-feature-discovery/030-nic-feature-discovery-ds.yaml

```diff
@@ -39,9 +39,9 @@ spec:
       serviceAccountName: nic-feature-discovery
       {{- end }}
       tolerations:
-      {{- if .Tolerations }}
-        {{- .Tolerations | yaml | nindent 8 }}
-      {{- end }}
+        {{- if .Tolerations }}
+          {{- .Tolerations | yaml | nindent 8 }}
+        {{- end }}
```

Comment:
Was this change needed in the same way the one for nv-ipam-node was? I'm in the process of adding unit tests to cover this case, and I don't see this function panicking the same way as the NVIPAM one is.

For context my tests are here (still WIP)

Reply:
I think this is the PR? https://github.com/Mellanox/network-operator/pull/814/files

Feel free to close this one then :)

Reply:
Thanks for the quick response. I'd definitely prefer to merge your PR - I was just following up with tests. My PR is build on top of yours and it's WIP until this one merges.

Reply:
Thanks for adding tests